### PR TITLE
Parts list updated.

### PR DESCRIPTION
**Pull request by SubOptimal**
- some information has been refined
- the parts from the saved shopping list are added as single positions into the parts list

### CODE REVIEW

File: content/docs/hardware/parts/_index.en.md [hunk 1]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
-  </td>
+  <td colspan="3" align="center">see notes</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-crimpkontakt-buchse-xh-jst-xh-ckb-p185091.html">JST contact BXH-001T-P0.6</a></td>
+  <td colspan="3" align="center">7</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-crimpkontakt-buchse-xh-jst-xh-ckb-p185091.html">JST housing XHP-2</a></td>
+  <td colspan="3" align="center">6</td>
```
Comment (tobst) [inline content/docs/hardware/parts/_index.en.md:104]:
```suggestion
  <td colspan="3" align="center">3</td>
```

File: content/docs/hardware/parts/_index.en.md [hunk 2]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
 <tr>
   <td>Passive and various components</td>
-  <td colspan="3" align="center">
-    Purchase according to <a href="https://www.reichelt.de/my/1746485">this list</a>
-  </td>
+  <td colspan="3" align="center">see notes</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-crimpkontakt-buchse-xh-jst-xh-ckb-p185091.html">JST contact BXH-001T-P0.6</a></td>
+  <td colspan="3" align="center">7</td>
```
Comment (tobst) [inline content/docs/hardware/parts/_index.en.md:100]:
```suggestion
  <td colspan="3" align="center">15</td>
```

File: content/docs/hardware/parts/_index.en.md [hunk 3]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-150-kohm-0207-250-mw-5--1-4w-150k-p1355.html">resistor (CFR) 150 kΩ, 250 mW, 5 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-300-kohm-0207-250-mw-5--1-4w-300k-p1407.html">resistor (CFR) 300 kΩ, 250 mW, 5 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.conrad.de/de/p/tru-components-tc-1206s4f1101t5e203-dickschicht-widerstand-1-1-k-smd-1206-0-25-w-1-1-st-tape-cut-1584104.html">SMD resistor 1.1 kΩ, 1206, 0.25 W, 1 %</a></td>
+  <td colspan="3" align="center">1</td>
```
Comment (tobst) [inline content/docs/hardware/parts/_index.en.md:136]:
```suggestion
  <td colspan="3" align="center">2</td>
```
New design will feature also THT-version for R6+7

**Review by tobst — CHANGES_REQUESTED** [review]

File: content/docs/hardware/parts/_index.en.md [hunk 4]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
-    Purchase according to <a href="https://www.reichelt.de/my/1746485">this list</a>
-  </td>
+  <td colspan="3" align="center">see notes</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-crimpkontakt-buchse-xh-jst-xh-ckb-p185091.html">JST contact BXH-001T-P0.6</a></td>
+  <td colspan="3" align="center">15</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-crimpkontakt-buchse-xh-jst-xh-ckb-p185091.html">JST housing XHP-2</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:103]: This is the wrong link.

File: content/docs/hardware/parts/_index.en.md [hunk 5]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+<tr>
+  <td><a href="https://www.reichelt.de/jst-stiftleiste-gerade-1x2-polig-xh-jst-xh2p-st-p185073.html">JST shrouded header B2B-XH-A</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/jst-stiftleiste-gerade-1x5-polig-xh-jst-xh5p-st-p185076.html">JST shrouded header B5B-XH-A</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-10-kohm-0207-250-mw-5--1-4w-10k-p1338.html">resistor (CFR) 10 kΩ, 250 mW, 5 %</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:123]: Don't have to be CFR, metal oxide or any other type fine as well. Just get the through-hole 1/4W.

Please capitalize the first word in every cell.

File: content/docs/hardware/parts/_index.en.md [hunk 6]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+<tr>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-150-kohm-0207-250-mw-5--1-4w-150k-p1355.html">resistor (CFR) 150 kΩ, 250 mW, 5 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-300-kohm-0207-250-mw-5--1-4w-300k-p1407.html">resistor (CFR) 300 kΩ, 250 mW, 5 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.conrad.de/de/p/tru-components-tc-1206s4f1101t5e203-dickschicht-widerstand-1-1-k-smd-1206-0-25-w-1-1-st-tape-cut-1584104.html">SMD resistor 1.1 kΩ, 1206, 0.25 W, 1 %</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:135]: Starting PCB 00.03.12 these should be 2x THT as well. Maybe add a note already so people can choose?

File: content/docs/hardware/parts/_index.en.md [hunk 7]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+  <td colspan="3" align="center">2</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/vielschicht-keramikkondensator-100n-10--x7r-2-5-100n-p22853.html">ceramic capacitor 100 nF, 10 %</a></td>
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/elko-radial-22-uf-10-v-1000-h-low-esr-aec-q200-rad-fc-22-10-p84587.html">radial electrolytic capacitor 22 µF, 10 V</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:143]: I think  "Low ESR" should be part of the specification here. AFAIK they are used as medium-size decoupling caps for the modules, so I think that needs low ESR. I'm not the engineer though. Low ESR usually isn't bad anyway, so even if it's not required, better safe than sorry?

File: content/docs/hardware/parts/_index.en.md [hunk 8]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+  <td colspan="3" align="center">2</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/schottkydiode-60-v-5-a-do-201ad-sb-560-p16081.html">Schottky diode, 60 V, 5 A</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:147]: `DO-201AD` should be included, or people might buy other packages that don't fit, or even SMD

Always consider what will happen if the link target goes 404.

File: content/docs/hardware/parts/_index.en.md [hunk 9]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/rueckstellende-sicherungen-max-40a-30v-6-6s-pfra-110-p35211.html">resettable fuse, max. 40 A - 30 V, 6.6 sec</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:151]: I sadly don't know what the package is called. It seems to be "radial" (considering it is mounted vertically, like the caps, not horizontally, like the axial resistors), and we could specify the pitch.

File: content/docs/hardware/parts/_index.en.md [hunk 10]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/platinensteckverbinder-gerade-weiss-2-polig-ps-25-2g-ws-p14825.html">PCB connector straight, 2-pin (pin and socket, hard wired)</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:155]: Pitch? 2.5mm or 2.54mm?

Also any other type of connector for the same pitch should work as well. Could add a note that this for soldering to the battery.

File: content/docs/hardware/parts/_index.en.md [hunk 11]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+  <td colspan="3" align="center">1</td>
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/de/de/36pol-stiftleiste-gewinkelt-rm-2-54-sl-1x36w-2-54-p19505.html">36-pin header, angled, pitch 2.54</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:159]: We need less than these 2 full connector strips. Especially in collective orders, you can order about a third and split them up.

* 4x angled for GPS
* 6x angled for SD-Card module
* 6x straight for charger
* optional: 5x straight for regulator (should be soldered SMD style though)

Also, some of the modules come with straight headers which get replaced by the angled ones. So you could reuse those for the modules that need straight headers and come with none.

But if you want to be sure, buy 10x angled and 11x straight.

In collective orders, give them a bit more, so maybe buy 36x or 40x of both angled and straight and split that into 3 parts, 12x-14x each. If you give them 12x, and the splitting goes wrong (sometimes it breaks in a bad way) there is a buffer to patch something together. Or pre-split into 4x and 6x size for the collective order, then you can fill 4 orders from a 40x angled strip.

File: content/docs/hardware/parts/_index.en.md [hunk 12]
```diff
@@ -89,16 +90,81 @@ checking everything is important.
...
+</tr>
+<tr>
+  <td><a href="https://www.reichelt.de/de/de/40pol-stiftleiste-gerade-rm-2-54-sl-1x40g-2-54-p19506.html">40-pin header, straight, pitch 2.54</a></td>
+  <td colspan="3" align="center">1</td>
 </tr>
 <tr>
-  <td>
-  16pin 2.54mm SMD connectors (<a href="https://www.ebay.de/itm/10-pcs-SMD-Buchsenleiste-1x-16-polig-RM-2-54mm-NEU-BP/231930033721">10pcs</a>, <a href="https://www.ebay.de/itm/100-pcs-SMD-Buchsenleiste-1x-16-polig-RM-2-54mm-NEW/233091176275">100pcs</a>)
-  </td>
+  <td><a href="https://www.fischerelektronik.de/web_fischer/de_DE/$catalogue/fischerData/PR/BL_LP5SMD_/datasheet.xhtml?branch=Steckverbinder">15 pin 2.54mm SMD connectors (BL LP 5 SMD)</a></td>
```
Comment (opatut) [inline content/docs/hardware/parts/_index.en.md:167]: Maybe add these notes:

* 15 pin is hard to find, you can also buy longer and cut to size.
* Same-side works too, we have a flexible footprint, but even-odd is more stable when soldered.

**Review by opatut — CHANGES_REQUESTED** [review]
Hi there. Lots of suggestions for extra information that I know from the top of my head. There is probably more.

Only some parts I wrote *need* changing (e.g. the broken link), the rest is suggestions that you can just ignore ;)

File: content/docs/hardware/parts/_index.en.md (hunk 7 above)
Comment (opatut) [reply content/docs/hardware/parts/_index.en.md:143]: Ping @Halbelunge, is low ESR required?

**Comment by SubOptimal** [comment]
The changes I pushed to https://github.com/SubOptimal/openbikesensor.github.io/tree/parts_list, but they are not shown here.
I opened a ticket with GitHub to get a bit of advice on what might have gone wrong.

File: content/docs/hardware/parts/_index.en.md (hunk 5 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: The link has been corrected.

File: content/docs/hardware/parts/_index.en.md (hunk 4 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: The link has been corrected.

File: content/docs/hardware/parts/_index.en.md (hunk 6 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: Hint for THT added for PCB's starting from version 00.03.12.

File: content/docs/hardware/parts/_index.en.md (hunk 7 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: Add requirement for 'low ERS' types.

File: content/docs/hardware/parts/_index.en.md (hunk 8 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: The specific type has been added.

File: content/docs/hardware/parts/_index.en.md (hunk 9 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: The type has been added.

File: content/docs/hardware/parts/_index.en.md (hunk 10 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: Pitch and usage have been added.

File: content/docs/hardware/parts/_index.en.md (hunk 11 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: Added note for the required number of pins, also for collective orders.

File: content/docs/hardware/parts/_index.en.md (hunk 12 above)
Comment (SubOptimal) [reply content/docs/hardware/parts/_index.en.md]: Added note for the possibility to buy longer SMD connectors and cut them.

**Comment by SubOptimal** [comment]
@opatut All suggestions have been incorporated.

**Review by opatut — APPROVED** [review]
Very good job, thanks for your diligent efforts.

**Pull request merged**